### PR TITLE
Migrate Voice traffic to AKAMAI

### PR DESCRIPTION
**Pull request by Koushik-Ayila**

### CODE REVIEW

**Comment by Koushik-Ayila** [comment]
@narayana-plivo, for the earlier commits we see the UT builds are not failing but for the recent ones they are giving an error `Fatal error: Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2". You are running 7.3.29. in /app/vendor/composer/platform_check.php on line 24` 
We have not made any changes in terms of Deployment Pipeline / UT config. On changing to composer v6, that was giving an access denied error though @huzaif-plivo has added permissions for version v6. 
As per [this](https://stackoverflow.com/questions/68433769/composer-dependencies-require-a-php-version-7-3-0?rq=1) reference, they suggested to add platform-check as false in composer.json. Can we proceed doing like that? Please let us know your thoughts.

**Review by narayana-plivo — CHANGES_REQUESTED** [review]
please clean up the retry code.
https://github.com/plivo/plivo-php/pull/248/files#diff-86d446eb96c277c1a3bfdf3e699a20daa98bb5d4637927830b827cabae5b5bf4R173 and isVoice request from all reference.

File: src/Plivo/BaseClient.php [hunk 1]
```diff
@@ -21,9 +21,9 @@ class BaseClient
      * @const BASE API URL
      */
     const BASE_API_URL = 'https://api.plivo.com/';
-    const VOICE_BASE_API_URL = 'https://voice.plivo.com/';
-    const VOICE_BASE_API_FALLBACK_URL_1 = 'https://voice-usw1.plivo.com/';
-    const VOICE_BASE_API_FALLBACK_URL_2 = 'https://voice-use1.plivo.com/';
+    const VOICE_BASE_API_URL = 'https://api.plivo.com/';
+    const VOICE_BASE_API_FALLBACK_URL_1 = 'https://api.plivo.com/';
```
[the remark this answers is not in the document]
Comment (Koushik-Ayila) [reply src/Plivo/BaseClient.php:25]: @narayana-plivo, We are retrying on the same url, hence isVoiceRequest and retry code should not be cleaned up.

[the remark this answers is not in the document]
Comment (narayana-plivo) [reply src/Plivo/BaseClient.php:25]: previously Voice url is https://voice.plivo.com/ and now you made https://api.plivo.com/. Its same as to BASE API URl. 
Why do you need two variable for the same value? 
similarly VOICE_BASE_API_FALLBACK_URL_1, VOICE_BASE_API_FALLBACK_URL_2? If the url is base url, why do need isVoiceRequest Check?

**Review by manish-plivo — APPROVED** [review]

**Review by narayana-plivo — APPROVED** [review]

**Pull request merged**